Example/Finjinon/ImagePickerAdapter.swift

```diff
@@ -31,8 +31,11 @@ open class ImagePickerControllerAdapter: NSObject, ImagePickerAdapter, UIImagePi
         return picker
     }
 
-    open func imagePickerController(_: UIImagePickerController, didFinishPickingMediaWithInfo info: [String: Any]) {
-        guard let referenceURL = info[UIImagePickerControllerReferenceURL] as? URL else {
+    open func imagePickerController(_: UIImagePickerController, didFinishPickingMediaWithInfo info: [UIImagePickerController.InfoKey: Any]) {
+// Local variable inserted by Swift 4.2 migrator.
+let info = convertFromUIImagePickerControllerInfoKeyDictionary(info)
```

Comment:
I think it's better to fix it without Swift 4.2 migrator.

Reply:
Yeah... wtf